src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -27,7 +27,7 @@ pub struct Allocation<Tag = (), Extra = ()> {
     /// at the given offset.
     relocations: Relocations<Tag>,
     /// Denotes which part of this allocation is initialized.
-    undef_mask: UndefMask,
+    uninit_mask: InitMask,
```

Comment:
As I said, the field name should also be adjusted to the type name: `init_mask`.

Reply:
The field name is still `uninit_mask`?